app/src/main/java/com/hedvig/app/feature/offer/OfferRepository.kt

```diff
@@ -33,6 +36,12 @@ class OfferRepository(
             .from(apolloClientWrapper.apolloClient.query(offerQuery).watcher())
     }
 
+    fun loadContracts(): Flow<Response<ContractStatusQuery.Data>> = apolloClientWrapper
+        .apolloClient
+        .query(ContractStatusQuery())
+        .watcher()
+        .toFlow()
```

Comment:
```suggestion
    fun loadContracts() = apolloClientWrapper
        .apolloClient
        .query(ContractStatusQuery())
        .toDeferred()
```

Reply:
Honestly though, I would just add this in `offer.graphql` and have it as part of the larger data